wd_helper.au3

```diff
@@ -538,9 +538,9 @@ EndFunc   ;==>_WD_IsWindowTop
 ; Description ...: Enter the specified frame.
 ; Syntax ........: _WD_FrameEnter($sSession, $vIdentifier)
 ; Parameters ....: $sSession    - Session ID from _WD_CreateSession
-;                  $vIdentifier - Index (as 0-based Integer) or Element ID (as String) or Null (Keyword)
+;                  $vIdentifier - Index (as 0-based Integer) or Element ID (as String) or Null (Keyword) or path in 'null/2/0' format
```

Comment:
please review this change in `$vIdentifier` description

Reply:
This change and changes to remarks are redundant. Have you considered doing something like this?

` ;                  $vIdentifier - Identifier of target frame. See remarks for formatting options`

Then you could place the full options in the remarks section.

Reply:
Plerase take a look on: [$vIdentifier description + remarks](https://github.com/Danp2/au3WebDriver/pull/364/commits/e451cd008902f6b8db546e45b9f9cc42dc97d2cb)

Reply:
> Identifier can be Null, specific element, or integer.

You took out vital details when you moved the text between Parameters and Remarks. Somehow we need to explain the difference between each of these options. We also need to be sure that any changes (ie: multiple line entry in either of the above secsions) appear correctly in the help file.

Reply:
So please propose complete remarks

Reply:
See https://github.com/Danp2/au3WebDriver/pull/364/commits/8c1dcc0c5f5b869615c27f0f972a524b7f371f83

Reply:
All is fine.